tasks/vm.py

```diff
@@ -0,0 +1,51 @@
+from invoke import task
+from .config import get_config
+import subprocess
+import os
+import invoke
+from .stack import stack_vm, get_scenario_folder, get_stack_name
+
+
+@task(help={
+    'install_agent': "Install the Agent (default True).",
+    'agent_version': "The version of the Agent for example '7.42.0~rc.1-1' or '6.39.0'",
+})
+def vm(ctx,
+       install_agent=True,
+       agent_version=""
+       ):
+    """
+    Create a new virtual machine on the cloud.
+    """
+    config = get_config()
+    aws_deploy(
+        config,
+        stack_vm,
+        install_agent,
+        f"ddagent:deploy={install_agent}",
+        f"ddagent:version={agent_version}",
+    )
+
+
+def aws_deploy(config, scenario_name, install_agent, *args):
```

Comment:
I think this belongs to another file

Reply:
I going to move it to another file when adding new command in a later PR.